### PR TITLE
feat: add custom chatThrottle

### PR DESCRIPTION
**Pull request by amitojsingh366**
- kousa: enforces `chatThrottle`
- kibbeh: adds rough UI to customize `chatThrottle`

i added this as per @goldyydev 's request. you should expect a pr from his side soon implementing this into the UI.

### CODE REVIEW

**Review by overlisted — APPROVED** [review]

**Review by dk-raw — APPROVED** [review]

File: kousa/lib/broth/message/user/get_info.ex [hunk 1]
```diff
@@ -43,6 +44,7 @@ defmodule Broth.Message.User.GetInfo do
       field(:numFollowers, :integer)
       field(:online, :boolean)
       field(:lastOnline, :utc_datetime_usec)
+      field(:lastChatMsg, :utc_datetime_usec)
```
Comment (benawad) [inline kousa/lib/broth/message/user/get_info.ex]: this shouldn't be stored in the db, look at how we throttle in chat genserver

**Review by benawad — CHANGES_REQUESTED** [review]

**Comment by benawad** [comment]
I also don't see any backend tests for the behavior

File: kousa/lib/broth/message/user/get_info.ex (hunk 1 above)
Comment (amitojsingh366) [reply kousa/lib/broth/message/user/get_info.ex]: done

**Comment by amitojsingh366** [comment]
> 
> 
> I also don't see any backend tests for the behavior

there are tests that update the `chatThrottle`, check `kousa/test/broth/_message/room/update_test.exs`

**Pull request merged**